src/PixiEditor.ChangeableDocument/Changeables/Graph/Nodes/Shapes/Data/EllipseData.cs

```diff
@@ -16,7 +16,14 @@ public EllipseData(VecD center, VecD radius)
     
     public override void Rasterize(DrawingSurface drawingSurface)
     {
-        using ChunkyImage img = new ChunkyImage(new VecI((int)Radius.X * 2, (int)Radius.Y * 2));
+        var imageSize = new VecI((int)Radius.X * 2, (int)Radius.Y * 2) + (VecI)Center;
```

Comment:
I don't get this, why Image size should have Center component added? Size of the circle is a radius*2